molecule/elasticsearch_7_10/molecule.yml

```diff
@@ -4,7 +4,22 @@ dependency:
 driver:
   name: docker
 platforms:
-  - name: elasticsearch-instance
+  - name: elasticsearch-ubuntu2404
```

Comment:
[Tutor](https://github.com/overhangio/tutor/blob/v18.1.3/tutor/templates/config/defaults.yml#L20) is using ES version 7.17.3, could we use that one instead of [this one](https://github.com/eduNEXT/atlas-ansible-utils/blob/main/roles/elasticsearch_7_10/defaults/main.yml#L18)? I think we should rename this role folder to `elasticsearch_7` given that it apparently can install any 7.x.x version with the variable `ELASTICSEARCH_VERSION` (please confirm that).